### PR TITLE
Log binary sizes

### PR DESCRIPTION
**Pull request by danieljurek**
Supersedes #3253

Draft PR to validate CI behavior

### CODE REVIEW

**Comment by danieljurek** [comment]
@antkmsft  -- Here is a PR with build failures for the UWP build.

File: eng/scripts/Get-BinarySizes.ps1 [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+        $metricLogJson = ConvertTo-Json $metricLogObject -Depth 2 -Compress
+        Write-Host "logmetric: $metricLogJson"
+    }
+}
+$binaries `
+    | Format-Table -Property Name, @{Name="SizeInKB"; Expression={"{0:N2}" -f ($_.Length / 1KB)}; Alignment='right'} `
+    | Out-String `
+    | Write-Host 
+
+return $binaries
```
Comment (danieljurek) [inline eng/scripts/Get-BinarySizes.ps1:43]: Newline

**Comment by antkmsft** [comment]
@danieljurek, I can find no easy solution for this at this point. I we spent way more time than expected trying to fix what should be an easy issue. Two fixes that were pioneered in this PR already went in to `main`, yet they are still not enough for the tests to run. If this is still important, let's create a work item, allocate time for it, prioritize, and figure out this problem.
You can close this PR now, and please let me know if you want me to remove all the failed attempts from this branch (if you don't plan to abandon it).

**Comment by danieljurek** [comment]
Closing this in favor of another PR to report metrics.

**Pull request closed without merging**